### PR TITLE
feat(app-shell): add launchdarkly tracking subgroup

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

This pull request _prematurely_ adds the `launchdarklyTrackingsubgroup`.

#### Description

This is still under development in the API. I wanted to see what parts have to change and get some early feedback.

### CODE REVIEW

File: schemas/mc.json [hunk 1]
```diff
@@ -6081,4 +6093,4 @@
       "timestamp": "Tue Dec 31 2019 16:44:28 GMT+0100 (Central European Standard Time)"
```
Comment (tdeekens) [inline schemas/mc.json]: I will pull this once deployed.

**Comment by tdeekens** [comment]
This is non customer facing but a feature for us as the MC. I am not sure if we want to mark it as chore? WDYT? :)

**Comment by tdeekens** [comment]
@adnasa and @emmenko this is now ready for review :)

**Comment by tdeekens** [comment]
> Do you know what changed here?

Not really. I just pulled the schema down. I think what changes is the "custom part" of the schema for the test project. So e.g. what custom types are created. Just a guess after looking into the file locally.

**Comment by emmenko** [comment]
Ok thanks for checking

**Pull request merged**